src/main/java/graphql/nadel/DefinitionRegistry.java

```diff
@@ -46,29 +45,39 @@ public SchemaDefinition getSchemaDefinition() {
     }
 
     public Map<Operation, List<ObjectTypeDefinition>> getOperationMap() {
-        return Stream.of(Operation.values()).collect(HashMap::new, (m, v) -> m.put(v, getOpsDefinitions(v.getName(), v.getDisplayName())), HashMap::putAll);
+        return Stream.of(Operation.values()).collect(HashMap::new, (m, v) -> m.put(v, getOpsDefinitions(v)), HashMap::putAll);
     }
 
     public List<ObjectTypeDefinition> getQueryType() {
-        return getOpsDefinitions(Operation.QUERY.getName(), Operation.QUERY.getDisplayName());
+        return getOpsDefinitions(Operation.QUERY);
     }
 
     public List<ObjectTypeDefinition> getMutationType() {
-        return getOpsDefinitions(Operation.MUTATION.getName(), Operation.MUTATION.getDisplayName());
+        return getOpsDefinitions(Operation.MUTATION);
     }
 
-    private List<ObjectTypeDefinition> getOpsDefinitions(String typeName, String typeDisplay) {
+    private List<ObjectTypeDefinition> getOpsDefinitions(Operation operation) {
```

Comment:
This change refactors `getOpsDefinitions` into `getOpsDefinitions` and `getOperationTypeName`. I use `getOperationTypeName` in `OverallSchemaGenerator`.